modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker_{context}"]
-= Creating a broker manually
+= Creating a Broker as a developer
```

Comment:
I actually wonder if it makes sense to document this currently for developers separately.
It's true that they can create a broker this way now, but they still can't delete them without cluster admin permissions.
I'm not sure how to explain the difference in using annotation for this for an admin vs a developer - should / can these just be one process, and instead we have a note that says anyone can create them this way but you need to be a cluster admin to delete brokers if they were created using annotation?

@devguyio @matzew @lberk I'd appreciate some input whoever has a minute first :slightly_smiling_face: thanks

Reply:
On top of that creating broker via annotations either on trigger or namespace has been removed/disabled on latest knative versions. So it may turns yup it's short lived feature. That needs confirmation from eventing team, though.

Reply:
there is an optional component:

https://github.com/knative/eventing/tree/release-0.17/config/sugar


if we want...... we can install it.  

for 0.16 the labeling etc still works OOTB

Reply:
Note that the `knative-eventing-injection` syntax is, however, deprecated:

https://github.com/knative/eventing/blob/release-0.16/pkg/reconciler/sugar/labels.go#L23

Reply:
Cool @matzew. Good to know about that optional part. But, I must dissagree with you about OOTB part. I've tested that, even today, for 0.16.1 Knative, installed with https://github.com/knative/eventing/releases/download/v0.16.1/eventing.yaml and this wasn't working.

```
$ kubectl apply -f https://github.com/knative/eventing/releases/download/v0.16.1/eventing.yaml
namespace/knative-eventing created
[..] CUT CUT
deployment.apps/imc-dispatcher created

$ kubectl get pods -n knative-eventing
NAME                                    READY   STATUS    RESTARTS   AGE
eventing-controller-7894f685d9-xczjx    1/1     Running   0          100s
eventing-webhook-566b77bb6b-pktdz       1/1     Running   1          100s
imc-controller-75d5dcc8f8-5pwn5         1/1     Running   0          99s
imc-dispatcher-788f4d84dd-r67xc         1/1     Running   0          98s
mt-broker-controller-84dcb5c76f-2f7xm   1/1     Running   0          99s
mt-broker-filter-66b78646cd-76gr5       1/1     Running   0          99s
mt-broker-ingress-7c6c8b6666-jxlnr      1/1     Running   0          99s

$ kubectl label namespace demo eventing.knative.dev/injection=enabled
namespace/demo labeled

$ kubectl get broker -n demo
No resources found in demo namespace.

$ kubectl get namespace demo -o json | jq '.metadata.labels'
{
  "eventing.knative.dev/injection": "enabled"
}
```

So, out of the box that isn't working.